MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include microcosm_eventsource *.ddl
```

Comment:
DDL files are now included as text in the dist

Reply:
How are these expected to be accessed?

Reply:
Ah, I see the resource_string call below.